From: SwampRabbit <swamprabbit@mxlinux.org>
Description: change prefix and ensure files get installed in the proper locations
Last-Update: 2020-09-24

--- a/Makefile 2021-09-024 10:04:37.000000000 -0400
+++ b/Makefile 2021-09-024 10:06:42.000000000 -0400
@@ -1,6 +1,6 @@
 LDLIBS   ?= -lpulse -lboost_program_options
 CXXFLAGS ?= -std=c++11 -Wall -Werror -Wextra -pedantic
-PREFIX   ?= /usr/local
+PREFIX   ?= /usr/
 RM       ?= rm -f
 
 target   := pamixer
@@ -28,6 +28,6 @@
 install: pamixer
 	install -d $(DESTDIR)$(PREFIX)/bin/
 	install $(target) $(DESTDIR)$(PREFIX)/bin/
-	install -d $(DESTDIR)$(PREFIX)/man/man1/
-	install $(manpage) $(DESTDIR)$(PREFIX)/man/man1/
-	gzip $(DESTDIR)$(PREFIX)/man/man1/$(manpage)
+	install -d $(DESTDIR)$(PREFIX)/share/man/man1/
+	install $(manpage) $(DESTDIR)$(PREFIX)/share/man/man1/
+	gzip $(DESTDIR)$(PREFIX)/share/man/man1/$(manpage)
